src/readinglistbar.cpp

```diff
@@ -59,14 +69,44 @@ void ReadingListBar::setupList()
         }
     }
 }
+// Receives single clicks
+void ReadingListBar::onItemClicked(QListWidgetItem* item)
+{
+    clickKind = 1;
+    QTimer::singleShot(QApplication::doubleClickInterval(), [=]() { // Give time for a double click be handled
+        if (clickKind == 1) {
+            openUrl(item, false);
+        }
+    });
+}
+// Receives double clicks
+void ReadingListBar::onItemDoubleClicked(QListWidgetItem* item)
+{ 
+    clickKind = 2;
+    openUrl(item, true);
+}
+// Receives single and middle click
+void ReadingListBar::onItemPressed(QListWidgetItem* item, Qt::MouseButtons buttons)
+{
+    if (buttons & Qt::MiddleButton) {
+        openUrl(item, true);
+    }
+}
+// Receives left clicks and activation key
+void ReadingListBar::onItemActivated(QListWidgetItem* item, Qt::MouseButtons buttons)
+{
```

Comment:
Add empty lines after the closing braces of function bodies.

Reply:
Yeah I was wondering on that